config/examples/stm32f407-discovery.config

```diff
@@ -9,3 +9,8 @@ WOLFBOOT_SECTOR_SIZE?=0x20000
 WOLFBOOT_PARTITION_BOOT_ADDRESS?=0x08020000
 WOLFBOOT_PARTITION_UPDATE_ADDRESS?=0x08040000
 WOLFBOOT_PARTITION_SWAP_ADDRESS?=0x08060000
+CLOCK_SPEED?=160000000
```

Comment:
This file rename will need to be adjusted in several places (docs and test scripts) to resolve build errors.